planning/behavior_velocity_planner/src/scene_module/blind_spot/debug.cpp

```diff
@@ -175,42 +91,44 @@ visualization_msgs::msg::MarkerArray BlindSpotModule::createDebugMarkerArray()
   visualization_msgs::msg::MarkerArray debug_marker_array;
 
   const auto state = state_machine_.getState();
-  const auto current_time = this->clock_->now();
+  const auto now = this->clock_->now();
 
   appendMarkerArray(
-    createPathMarkerArray(debug_data_.path_raw, "path_raw", lane_id_, 0.0, 1.0, 1.0),
-    &debug_marker_array, current_time);
+    debug::createPathMarkerArray(
+      debug_data_.path_raw, "path_raw", lane_id_, now, 0.3, 0.0, 0.0, 0.0, 1.0, 1.0),
```

Comment:
@takayuki5168 
this will cause scale error in rviz please consider this value same as intersection.
```suggestion
      debug_data_.path_raw, "path_raw", lane_id_, now, 0.6, 0.3, 0.3, 0.0, 1.0, 1.0),
```

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/1773/commits/22359b11806fcd5ea5ed5e669395f83ae2eb0cfc
DONE